src/org/thoughtcrime/securesms/components/ThumbnailView.java

```diff
@@ -290,18 +290,6 @@ protected void onPostExecute(Boolean success) {
     return result;
   }
 
-  public ListenableFuture<Boolean> setImageResource(@NonNull GlideRequests glideRequests, @NonNull Uri uri) {
-    SettableFuture<Boolean> future = new SettableFuture<>();
-
-    glideRequests.load(new DecryptableUri(uri))
-                 .diskCacheStrategy(DiskCacheStrategy.NONE)
-                 .transforms(new CenterCrop(), new RoundedCorners(radius))
```

Comment:
Removing this unused function has nothing to do with my PR, but it felt better to have removed all `CenterCrop` from this file :)